src/allow_list.rs

```diff
@@ -0,0 +1,37 @@
+use anyhow::Result;
+use ipfs_embed::Cid;
+use libipld::multibase::Base;
+use reqwest::{get, StatusCode};
+use serde::{Deserialize, Serialize};
+use ssi::did::DIDURL;
+
+#[rocket::async_trait]
+pub trait OrbitAllowList {
+    async fn is_allowed(&self, oid: &Cid) -> Result<Vec<DIDURL>>;
+}
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct OrbitAllowListService {
+    pub api: String,
+}
+
+impl Default for OrbitAllowListService {
+    fn default() -> Self {
+        Self {
+            api: "http://localhost:11000".into(),
+        }
+    }
+}
+
+#[rocket::async_trait]
+impl OrbitAllowList for OrbitAllowListService {
+    async fn is_allowed(&self, oid: &Cid) -> Result<Vec<DIDURL>> {
```

Comment:
From our discussions about the different kinds of allow lists, I think the API should take the DIDURL we're interested in and tell us if it's part of the controllers?